pkg/ignition/builder.go

```diff
@@ -71,6 +71,15 @@ func (b *ignitionBuilder) Generate() ([]byte, error) {
 		})
 	}
 
+	config.Storage.Files = append(config.Storage.Files, ignitionFileEmbed(
+		"/etc/NetworkManager/conf.d/clientid.conf",
+		0644, false,
+		[]byte("[main]\ndhcp=dhclient\n[connection]\nipv6.dhcp_duid=ll")))
```

Comment:
There's no dhclient in RHCOS anymore - so I think this needs to match what's in the installer/MCO not the previous IPA image:

https://github.com/openshift/installer/blob/master/data/data/bootstrap/baremetal/files/etc/NetworkManager/conf.d/99-baremetal.conf
https://github.com/openshift/machine-config-operator/blob/master/templates/common/on-prem/files/NetworkManager-onprem.conf.yaml#L9..L10